front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -15,155 +15,191 @@ import messages from '../../messages';
 // typings
 import { ParticipationMethod } from 'services/participationContexts';
 import { ApiErrors } from '..';
+import { getMethodConfig } from 'utils/participationMethodUtils';
+import { isNilOrError } from 'utils/helperUtils';
+import { IPhase } from 'services/phases';
+import { IProjectData } from 'services/projects';
 
 interface Props {
-  participation_method: string;
+  participation_method: ParticipationMethod;
+  phase?: IPhase | undefined | null;
+  project?: IProjectData | undefined | null;
   showSurveys: boolean;
   apiErrors: ApiErrors;
   handleParticipationMethodOnChange: (
     participation_method: ParticipationMethod
   ) => void;
 }
 
-export default ({
+export const ParticipationMethodPicker = ({
   participation_method,
   showSurveys,
   apiErrors,
+  phase,
+  project,
   handleParticipationMethodOnChange,
-}: Props) => (
-  <SectionField>
-    <SubSectionTitle>
-      <FormattedMessage {...messages.participationMethodTitleText} />
-      <IconTooltip
-        content={<FormattedMessage {...messages.participationMethodTooltip} />}
-      />
-    </SubSectionTitle>
-    <ParticipationMethodRadio
-      onChange={handleParticipationMethodOnChange}
-      currentValue={participation_method}
-      value="ideation"
-      name="participationmethod"
-      id="participationmethod-ideation"
-      label={
-        <LabelHeaderDescription
-          header={<FormattedMessage {...messages.inputAndFeedback} />}
-          description={
-            <FormattedMessage {...messages.inputAndFeedbackDescription} />
-          }
-        />
-      }
-    />
+}: Props) => {
+  const chooseParticipationMethod = () => {
+    if (!isNilOrError(phase)) {
+      return phase.data.attributes.participation_method;
+    }
+    if (!isNilOrError(project)) {
+      return project.attributes.participation_method;
+    }
+    // Before a new project or phase is saved, use ideation as a default
+    // fallback config to control the radio behaviour.
+    return 'ideation';
+  };
 
-    <FeatureFlag name="participatory_budgeting">
-      <ParticipationMethodRadio
-        onChange={handleParticipationMethodOnChange}
-        currentValue={participation_method}
-        value="budgeting"
-        name="participationmethod"
-        id={'participationmethod-budgeting'}
-        label={
-          <LabelHeaderDescription
-            header={
-              <FormattedMessage
-                {...messages.conductParticipatoryBudgetingText}
-              />
-            }
-            description={
-              <FormattedMessage
-                {...messages.conductParticipatoryBudgetingDescriptionText}
-              />
+  const config = getMethodConfig(chooseParticipationMethod());
+  return (
+    <SectionField>
+      <SubSectionTitle>
+        <FormattedMessage {...messages.participationMethodTitleText} />
+        {!config.methodIsLocked && (
```

Comment:
Something minor. For booleans, it is better to name them starting with is, can, has, should. It makes it easy to point them out

Reply:
Ah true - ok I'll update it :) Thanks!